src/types.ts

```diff
@@ -262,7 +284,9 @@ export type GARFunctions =
   | 'finalizeOperatorStakeDecrease'
   | 'updateGatewaySettings';
 
-export type FoundationFunctions = 'gateway' | 'updateState' | 'mint';
+export type FoundationFunctions =
+  | 'initiateFoundationAction'
+  | 'signFoundationAction';
```

Comment:
how does this fit into the `tick the state` approach we're pursuing with auctions (and ideally all of these multi step interactions)